changelog 09 13 2024 (#2154)
neondatabase/website: content/changelog/2024-09-13.md

---
description: tbd
---

### This week’s changelog – Live from Lisbon!

![Neon Team Portugal](/docs/relnotes/neon_team.jpg)

> Our team is offsite this week in the beautiful city of Lisbon, planning what’s next for Neon as we close out an exciting Q3 and start looking forward to the next quarter and beyond. It's been an incredible opportunity to reconnect with colleagues, welcome new team members, meet Neon customers, and collaborate on the next phase of Neon's journey.

Reviewer: ```suggestion
Our team is in Lisbon this week, wrapping up Q3 and planning for what's next. It's been a great opportunity to reconnect, welcome new colleagues, meet with customers, and map out the next steps for Neon. 
```

> While we're enjoying the chance to meet face-to-face, we thought we would take the opportunity to recap some of the key items we released over the past few months:

Reviewer: ```suggestion
As we look ahead as a team, here's a quick look back at some key features we've released over the past few months:
```

> - **Inbound Logical Replication**: Enables low-downtime migrations to Neon, making it easy to use Neon as a development environment with features like branching and [GitHub integration](/docs/guides/neon-github-integration), and supports migrations from other providers and between Neon projects. Check out our guides on [Replicating Data to Neon](/docs/guides/logical-replication-guide#replicate-data-to-neon) and [Project-to-Project Replication](/docs/guides/logical-replication-neon-to-neon).

Reviewer: ```suggestion
- **Inbound Logical Replication**: Enables low-downtime migrations to Neon, making it easy to use Neon as a development environment with features like branching and [GitHub integration](/docs/guides/neon-github-integration). It also supports migrations from other providers and between Neon projects. Check out our guides on [Replicating Data to Neon](/docs/guides/logical-replication-guide#replicate-data-to-neon) and [Project-to-Project Replication](/docs/guides/logical-replication-neon-to-neon).
```

> - **Organizations**: Now available for Early Access users. Create and manage your organization's projects and teams with ease. Join our [Early Access Program](https://console.neon.tech/app/settings/early-access) and refer to the [Organizations docs](/docs/manage/organizations) for more information.

Reviewer: ```suggestion
- **Organizations**: Now available for Early Access users. Create and manage your organization's projects and teams from a single account. Join our [Early Access Program](https://console.neon.tech/app/settings/early-access) and refer to the [Organizations docs](/docs/manage/organizations) for more information.
```

- **Autoscaling GA**: Neon's Autoscaling feature is now generally available, automatically adjusting resources as needed. Read the announcement [here](https://neon.tech/blog/neon-autoscaling-is-generally-available) and explore the [Autoscaling docs](/docs/introduction/autoscaling).
- **Autoscaling on Free Plan**: Free Plan users can now experience Neon's Autoscaling, within reasonable limits. Learn more in the [Autoscaling Guide](/docs/guides/autoscaling-guide).
- **A New Business Plan**: Offers 500 GiB of storage and 1,000 compute hours, with potential cost savings for customers needing more storage than offered by our Scale plan. Learn more on our [Pricing](https://neon.tech/pricing) and [Plans](/docs/introduction/plans) pages.

> For a peek at what we'll be delivering next, check out the [Neon roadmap](/docs/introduction/roadmap). You can expect to see some additions here as soon as we finish fine-tuning our Q4 plan. Stay tuned!

Reviewer: Maybe remove this paragraph about the roadmap entirely?

Author: removed.
